src/shadowbox/model/access_key.ts

```diff
@@ -66,6 +66,8 @@ export interface AccessKeyRepository {
   removeAccessKey(id: AccessKeyId): boolean;
   // Lists all existing access keys
   listAccessKeys(): AccessKey[];
+  // Changes the default port for new access keys.
+  setDefaultPort(port: number): Promise<void>;
```

Comment:
Rename to `setPortForNewAccessKeys`, for consistency with the config and API.

`defaultPort` is too ambiguous. It doesn't communicate whether it affects the management API or the existing access keys.